metron-platform/metron-common/README.md

```diff
@@ -167,6 +172,14 @@ The `!=` operator is the negation of the above.
 | [ `WEEK_OF_YEAR`](#week_of_year)                                                                   |
 | [ `YEAR`](#year)                                                                                   |
 
+### `APPEND_IF_MISSING`
+  * Description: Appends the suffix to the end of the string if the string does not already end with any of the suffixes.
+  * Input:
+    * string - The string to be appended.
+    * suffix - The string suffix to append to the end of the string.
+    * suffixes - Optional - Additional string suffixes that are valid terminators.
```

Comment:
Ah, got it.  The documentation you point at for apache commons PrependIfMission method actually uses signature:
```
CharSequence... suffixes
```
so it is a varargs usage.  You can do the same with Stellar, or we have found it better to allow a list of strings instead (which you would explode into the calling varargs in the java code).  Using a list instead of varargs makes it much easier to extend the function later if we choose.

It's up to you whether to do this now.  We are trying to close a RC, so there is some time pressure.  Maybe you should just open a jira to enhance this later with a list arg.  

The current implementation of your call to PrependIfMissing is such that the "additionalsuffix" description you suggest would be correct.